src/App.jsx

```diff
@@ -10,11 +10,22 @@ function App() {
     setTodoList([...todoList, newTodo])
   }
 
+  function completeTodo(id) {
+    const updatedTodos = todoList.map((todo) => {
+      if (todo.id === id) {
+        let value = !todo.isCompleted; // it's prepared for a future uncheck function
```

Comment:
This works however since the intention is to always have `isCompleted` marked as `true` for todo items that match the passed `id`, it's best to do just that. Although unlikely, this could create some bugs.